server/views/components/figure/index.njk

```diff
@@ -18,8 +18,10 @@
       {%- endfor %}"
       alt="{{ caption }}" />
   {% endif %}
-  <figcaption class="figure__caption">
-    {% component 'icon', {icon: icon, extraClasses: ['figure__icon']} %}{% endcomponent %}
-    {{ caption }}
-  </figcaption>
+    {% if full %}<div class="container">{% endif %}
```

Comment:
Yep, there are captions on all images – this conditional determines whether or not the captions should live in a `container` element (the images are full bleed, but the captions should still be constrained within the bounds of the grid).